ApolloCodegen/Sources/ApolloCodegen/main.swift

```diff
@@ -72,6 +72,17 @@ struct SwiftScript: ParsableCommand {
                                   options: codegenOptions)
         }
     }
+
+    struct DownloadSchemaAndGenerateCode: ParsableCommand {
+        static var configuration = CommandConfiguration(
+            commandName: "all",
+            abstract: "Downloads the schema and generates swift code.")
+
+        func run() throws {
+            try! DownloadSchema().run()
+            try! GenerateCode().run()
```

Comment:
These should be `try` not `try!`, as they are in a throwing function. The function will rethrow with `try`, whereas the whole app will crash with `try!`.

Reply:
True, can change